config/wcp/vmoperator/manager_env_var_patch.yaml

```diff
@@ -52,6 +52,12 @@
     name: FSS_WCP_TKG_Multiple_CL
     value: "<FSS_WCP_TKG_Multiple_CL_VALUE>"
 
+- op: add
+  path: /spec/template/spec/containers/0/env/-
+  value:
+    name: FSS_WCP_VMSERVICE_K8S_WORKLOAD_MGMT_API
+    value: "<FSS_WCP_VMSERVICE_K8S_WORKLOAD_MGMT_API_VALUE>"
+
```

Comment:
Should these be added to the `config/local/vmoperator/local_env_var_patch.yaml file` as well for consistency?